Henry Zhang/week01/homework_day03/js/exercises.js

```diff
@@ -0,0 +1,114 @@
+// # Array and Functions Bonus Material
+//
+// 1. Define a function `maxOfTwoNumbers` that takes two numbers as arguments and returns the largest of them. Use the if-then-else construct available in Javascript. You'll have to remember your pre-work, or do some googling to figure this out.
+
+const maxOfTwoNumbers =function (a, b) {
+  if (a > b){
+    return a;
+  }
+    return b;
+}
+
+console.log(`${maxOfTwoNumbers(4,56)}`);
+
+
+// 2. Define a function `maxOfThree` that takes three numbers as arguments and returns the largest of them.
+
+const maxOfThree = function (a, b, c) {
+    let larger = a >b ? a:b;
+    let max = larger >c? larger : c;
+
+      return max;
+}
+
+console.log(maxOfThree(2,15,7));
+
+// 3. Write a function that takes a character (i.e. a string of length 1) and returns true if it is a vowel, false otherwise.
+const isVowel =function (str) {
+    for (var i = 0; i < str.length; i++) {
+      if (str[i]==="a") {
+        return 'true';
+
+      }else if (str[i]==='e'){
+        return 'true';
+
+      }else if (str[i]==='i'){
+        return 'true';
+
+      }else if (str[i]==='o'){
+        return 'true';
+
+      }else if (str[i]==='u'){
+        return 'true';
+      }
+      return 'false';
+      }
+
+    }
```

Comment:
I think you've over complicated this by creating a loop. The question states that the input will only be a single character, so you don't need to loop through the input because it is one. 
Also please keep in mind spacing - it will make this a lot more readable. If I were to make a couple of edits to this, I'd do: 
const isVowel = function (char) {
      if (char === "a" || char === "e" || char === "i" || char === "o" || char === "u") {
        return true;
      } else {
        return false;
      }
}

Check out the [GitBook](https://app.gitbook.com/@cjcdc/s/seic38/) for some information around Booleans. You don't need `''` around true and false, because they're actually a datatype called Booleans.